templates/hoover.nomad

```diff
@@ -87,6 +87,40 @@ job "hoover" {
     }
   }
 
+  group "tika" {
+    task "tika" {
+      driver = "docker"
+      config {
+        image = "logicalspark/docker-tikaserver:1.20"
+        port_map {
+          tika = 9998
+        }
+        labels {
+          liquid_task = "tika"
+        }
+      }
+      resources {
+        memory = 800
+        cpu = 200
+        network {
+          port "tika" {}
+        }
+      }
+      service {
+        name = "tika"
```

Comment:
I think you should update the collections template to point to the new service name for tika ;)

Reply:
Yep.